api/http/saml.go

```diff
@@ -60,9 +60,9 @@ type SamlSLORequestHandler struct {
 
 // ServeHTTP is the initial entry point for authentication.
 func (service SamlSLORequestHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
-	if !service.Env.True(api.SamlEnabled) {
+	if !service.Env.True(api.SamlSloEnabled) {
```

Comment:
In the absence of something at start up making sure SAML_ENABLED and SAML_SLO_ENABLED are set logically consistently, we should add the SLO check to the existing SAML check instead of replacing it.